plugins/changed_config_files/changed_config_files_inspector.rb

```diff
@@ -37,25 +37,27 @@ def initialize(system, description)
   end
 
   def inspect(filter, options = {})
-    do_extract = options[:extract_changed_config_files]
+    do_extract = options[:extract_changed_changed_config_files]
     system.check_retrieve_files_dependencies if do_extract
 
     count = 0
     files = @system.managed_files_database.changed_files do |chunk|
       count += chunk.lines.count { |l| !l.chomp.end_with?(":") && l.split(" ")[1] == "c" }
-      Machinery::Ui.progress(" -> Found #{Machinery.pluralize(count, "%d changed config file")}...")
+      Machinery::Ui.progress(
+        " -> Found #{Machinery.pluralize(count, "%d changed configuration file")}..."
```

Comment:
Just as a comment I think this would read better if it said "changed configuration file(s)" but we can do that in a different PR otherwise this would be the only scope that reads this way